README.md

```diff
@@ -1,4 +1,27 @@
-# Azure Functions Bindings for Azure SignalR Service
+# Azure Functions Bindings for Azure SignalR Service  <!-- omit in toc -->
+
+## Content  <!-- omit in toc -->
+
+- [Build Status](#build-status)
+- [NuGet Packages](#nuget-packages)
+- [Intro](#intro)
+  - [Supported scenarios](#supported-scenarios)
+  - [Bindings](#bindings)
+  - [Current limitations](#current-limitations)
+- [Prerequisites](#prerequisites)
+- [Usage](#usage)
+  - [Create Azure SignalR Service instance](#create-azure-signalr-service-instance)
+  - [Create Function App with extension](#create-function-app-with-extension)
+  - [Add application settings](#add-application-settings)
+  - [Using the SignalRConnectionInfo input binding](#using-the-signalrconnectioninfo-input-binding)
```

Comment:
>[Using the SignalRConnectionInfo input binding] [](start = 4, length = 47)

uniform the title with output binding. Simplify to `SignalR input binding` or `Use SignalR input binding` like previous pattern of do something, same for next output binding title.